Contrib/mmpa/test_rfrag.py

```diff
@@ -0,0 +1,25 @@
+#
+#  Copyright (C) 2017 greg Landrum
+#
+#   @@ All Rights Reserved  @@
+#
+import unittest, subprocess, os
+from rdkit import RDConfig
+
+
+class TestCase(unittest.TestCase):
+  def test1Github1406(self):
+    with open('data/simple.smi') as inf:
+      p = subprocess.run(('python', 'rfrag.py'),stdin=inf,stdout=subprocess.PIPE)
```

Comment:
Instead of having everything in the `if __name__ == '__main__':` block, the code there can be moved into a function and that called from the test. No need to start a new subprocess. stdout and stderr can be captured using `rdkit.TestRunner.redirect_stdout` and `rdkit.TestRunner.redirect_stdout` (also available in `contextlib` from 3.4 onwards).